tests/functional/flows/test_start_flow.py

```diff
@@ -0,0 +1,122 @@
+import json
+import re
+
+from globus_sdk._testing import RegisteredResponse, load_response
+
+
+def test_start_flow_text_output(run_line, add_flow_login):
+    # Load the response mock and extract critical metadata.
+    response = load_response("flows.run_flow")
+    flow_id = response.metadata["flow_id"]
+    body = response.metadata["request_params"]["body"]
+    tags = response.metadata["request_params"]["tags"]
+    label = response.metadata["request_params"]["label"]
+    run_monitors = response.metadata["request_params"]["run_monitors"]
+    run_managers = response.metadata["request_params"]["run_managers"]
+    add_flow_login(flow_id)
+
+    # Configure identities.
+    owner_identity = {
+        "username": "yogi@jellystone.park",
+        "name": "Yogi",
+        "id": response.json["run_owner"].split(":")[-1],
+        "identity_provider": "c8abac57-560c-46c8-b386-f116ed8793d5",
+        "organization": "Hanna-Barbera",
+        "status": "used",
```

Comment:
```suggestion
        "status": "smarter-than-your-av-er-age-bear",
```

Reply:
I daren't change this lest it be used unexpectedly! :wink: